fixit/rules/gather_sequential_await.py

```diff
@@ -24,9 +24,10 @@ async def async_foo():
                 return await async_bar()
             """
         ),
-        # await in a loop is fine if it's a test.
         Valid(
```

Comment:
Assuming this is to address the generated documentation issue.
A test filename is provided for this test case.
We can simply also provide the file name in the generated doc to make it clear.

Reply:
Yep, that also works!